.travis.yml

```diff
@@ -0,0 +1,23 @@
+sudo: required
+dist: trusty
+
+language: python
+python:
+  - 3.5
+
+
+install:
+  # Install code sniffer.
+  - if [[ $TRAVIS_PYTHON_VERSION == 3.5 ]]; then
+    pip install pycodestyle;
+    pip install --upgrade pycodestyle;
+    source ~/.bash_profile;
+    source ~/.bashrc;
+    fi
+
+
+script:
```

Comment:
Same as before, every command in a different line, like:
```yml
script:
  - pycodestyle --show-source --max-line-length=119 --show-pep8 .
```

And remove `- if [[ $TRAVIS_PYTHON_VERSION == 3.5 ]]; then` and `fi`